libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/AlonzoBBODY.hs

```diff
@@ -15,6 +15,7 @@ module Test.Cardano.Ledger.Examples.AlonzoBBODY (tests) where
 
 import Cardano.Crypto.Hash.Class (sizeHash)
 import Cardano.Ledger.Address (Addr (..))
+import qualified Cardano.Ledger.Address as CLA (Withdrawals (..))
```

Comment:
```suggestion
import Cardano.Ledger.Address (Withdrawals (..))
```
Instead of introducing a clash of names in the same codebase and resort to qualified imports, it is better to avoid the  clash altogether.

So, either revert it back to `Wdrls` or use a tick `Withdrawals'`, which would align with others

Reply:
:bow: